query/parse.go

```diff
@@ -231,6 +231,15 @@ func regexpQuery(text string, content, file bool) (Q, error) {
 		return nil, err
 	}
 
+	if hasCapture(r) {
+		re, err := ConvertCapture(r)
+		if err == nil {
```

Comment:
nit: I would flip this around. Why? The "pattern" `if err!=nil` is so omnipresent in Go that we should always use it unless it makes the code much harder to read. WDYT?

Reply:
That condition is gone with the latest commit but the ordering of the if branches was something I originally debated in my head.
On one hand like you say the `if err != nil` pattern is well established. On the other the positive/success case, the one you actually want, occurs when `err == nil` so I think it more clearly conveys the intention if that comes first.
Going with `err != nil` probably would have been the more idiomatic choice though